portfolio/src/main/webapp/script.js

```diff
@@ -38,6 +42,28 @@ function createMap() {
       });
 }
 
+/**
+ * Draws a chart and adds it to the page.
```

Comment:
Can you clarify that it is hardcoded data at the moment? And if you are planning on changing that in the future, can you add a TODO?

Reply:
Done!